integration-tests/docker/middlemanager.conf

```diff
@@ -17,8 +17,8 @@ command=java
   -Ddruid.indexer.fork.property.druid.processing.buffer.sizeBytes=75000000
   -Ddruid.indexer.fork.property.druid.processing.numThreads=1
   -Ddruid.indexer.fork.server.http.numThreads=100
-  -Ddruid.s3.accessKey=AKIAIMKECRUYKDQGR6YQ
-  -Ddruid.s3.secretKey=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
+  -Ddruid.s3.accessKey=AKIAJI7DG7CDECGBQ6NA
```

Comment:
Just wonder what s3 storage is accessed by those credentials, is it safe to keep those credentials in `github` repo?

Reply:
This is not our s3 account
https://github.com/apache/incubator-druid/blob/master/integration-tests/docker/middlemanager.conf#L19

Reply:
I don't understand why we change them?

Reply:
Since we depend on third-party's s3 account in integration-tests (and it was updated). As a solution, we might duplicate that data into our s3.